### PR TITLE
added node 10 to travis and appveyor

### LINKED ISSUE #600: node 10.0.0 compatibility
Are there plans to support node 10.0.0 anytime soon?

### PR DESCRIPTION
**Pull request by 4ware**
Closes #600

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -1,46 +1,48 @@
...
+  - nodejs_version: "8"
+    platform: x64
+  - nodejs_version: "8"
+    platform: x86
+  - nodejs_version: "9"
+    platform: x64
+  - nodejs_version: "9"
+    platform: x86
+  - nodejs_version: "10"
+    platform: x86
```
Comment (agathver) [inline appveyor.yml]: NodeJS x64 ?

**Comment by agathver** [comment]
Thank you for your contribution @4ware!

**Comment by agathver** [comment]
NodeJS 4 has become EOL, so we can remove the versions from the CI matrix. This will allow the tests to pass.

**Comment by pauliusuza** [comment]
@agathver Could you please merge this and apply the changes? We are blocked from updating to node 10 because of this. Thanks

**Comment by agathver** [comment]
@pauliusuza,

I have been traveling for a few days and asked for changes on this one which hasn't been done yet.

We need if/else conditions like this for NodeJS 4, it doesn't support npm 6.
https://github.com/kelektiv/node.bcrypt.js/blob/v0.8.7/.travis.yml#L24

```
 - if [ "$TRAVIS_NODE_VERSION" = "4" ]; then npm install -g npm@5; else npm install -g npm@latest; fi;
```
And same in appveyor.yml

```
 - 'if "%nodejs_version%" == "4" (npm install -g npm@5) else (npm install -g npm@latest)'
```

If you make a quick PR, I would merge them if it passes CI, and trigger a binary publish.

>We are blocked from updating to node 10 because of this. Thanks

It _should not_ block your upgrade. The official instrcutions are different, pre-built binaries are just provided as a convenience. See https://github.com/kelektiv/node.bcrypt.js/wiki/Installation-Instructions

**Comment by pauliusuza** [comment]
@agathver install falls back to build, but fails. I described the error in #599. 
Will open a PR for build files

**Comment by agathver** [comment]
This has been merged as the part of #603

**Pull request closed without merging**